### PR TITLE
MIVisionX 1.9.5 Release Updates

### PR DESCRIPTION
**Pull request by kiritigowda**
* Readme Updates
* Fix Warnings
* Cleanup

### CODE REVIEW

File: tests/vision_tests/runVisionTests.py [hunk 1]
```diff
@@ -181,35 +179,56 @@ def write_formatted(output, f):
...
-    os.system('(cd gdfs; ./../'+runvx_exe_dir+' -frames:100 -affinity:' +
-              hardwareMode+' -dump-profile file '+testFileName+' | tee -a openvx_test_results/VisionOutput.log)')
-    print("\n")
+
+if testFilter == 0:
+    print("\nrunVisionTests - OpenVX Vision Tests V-"+__version__+"\n")
+    for i in range(len(visionTestConfig)):
+        testFileName = visionTestConfig[i]
+        print("Running Test Script: "+testFileName)
+        os.system('(cd gdfs; ./../'+runvx_exe_dir+' -frames:100 -affinity:' +
```
Comment (asalmanp) [inline tests/vision_tests/runVisionTests.py:229]: adding **./../** before runvx_exe_dir is causing runvx not found error. By removing it, I was able to run the script on UB18.04 fine by passing /opt/rocm/mivisionx/bin for the --runvx_directory argument.

**Review by asalmanp — CHANGES_REQUESTED** [review]
just some minor comments.

File: tests/vision_tests/runVisionTests.py (hunk 1 above)
Comment (kiritigowda) [reply tests/vision_tests/runVisionTests.py:229]: @asalmanp can you verify if the path to runvx is not /opt/rocm.. but a local dir path, will it raise an issue?

Comment (asalmanp) [reply tests/vision_tests/runVisionTests.py:229]: @kiritigowda I just verified that if the path to runvx is a local dir path, the original runVisionTests script will cause the same (runvx not found error), and can just be run by removing the ./../ before the runvx_exe_dir..

File: tests/vision_tests/runVisionTests.py [hunk 2]
```diff
@@ -181,35 +179,56 @@ def write_formatted(output, f):
...
+        nodeName, nodeFormat = openvxNodes[i]
+        print("   %-5d - %-30s\n" % ((i+1), nodeName))
+    exit()
+if runvxDir == '':
+    print("ERROR: RunVX Executable Directory Required")
+    exit()
 
 # RunVX Application
 runVX_exe = runvxDir+'/runvx'
 runvx_exe_dir = os.path.expanduser(runVX_exe)
```
Comment (kiritigowda) [inline tests/vision_tests/runVisionTests.py:220]: @asalmanp I saw that I already expand the path!

**Review by asalmanp — APPROVED** [review]
LGTM.

**Pull request merged**